parm/wflow/aqm_prep.yaml

```diff
@@ -142,17 +197,32 @@ task_aqm_ics:
         datadep_date_tag_tracer:
           attrs:
             age: 00:00:00:05
-          text: '<cyclestr offset="-{{- workflow.INCR_CYCL_FREQ -}}:00:00">&COMIN_DIR;</cyclestr>/RESTART/<cyclestr>@Y@m@d.@H@M@S.fv_tracer.res.tile1.nc</cyclestr>'
+          value:
+            - cyclestr:
+                attrs:
+                  offset: "-{{ workflow.INCR_CYCL_FREQ }}:00:00"
```

Comment:
It is a little early at this point, I think. We'd want to decide where a thing like this goes. Given that it's a Rocoto-specific string we need here, I started off trying to add it as a Rocoto entity, then I became concerned that a line with an entity reference in it wouldn't pass UW's schema. Here's an example:

```
cycledef:
    - attrs:
        group: at_start
      spec: '{{workflow.DATE_FIRST_CYCL}}00 {{workflow.DATE_FIRST_CYCL}}00 &CYCLE_FREQ;'
```

I could add it with a pyyaml reference, but it would still show up in several locations -- any of the parm/wflow/*yaml files that needed it. Putting it in a user-facing config doesn't seem right, either.

I prefer to table this one for a bit if that's okay.

Reply:
Welp, I forgot to stash my changes before running the next test and my fear was confirmed:

```
3 UW schema-validation errors found in Rocoto config
Error at workflow -> cycledef -> 0 -> spec:
  '202008100000 202008100000 &CYCLE_FREQ;' is not valid under any of the given schemas
Error at workflow -> cycledef -> 1 -> spec:
  '202008100000 202008100000 &CYCLE_FREQ;' is not valid under any of the given schemas
Error at workflow -> cycledef -> 2 -> spec:
  '202008100000 202008100000 &CYCLE_FREQ;' is not valid under any of the given schemas
```